requirements/dev.txt

```diff
@@ -509,6 +511,13 @@ django-statsd-mozilla==0.4.0 \
     --hash=sha256:0d87cb63de8107279cbb748caad9aa74c6a44e7e96ccc5dbf07b89f77285a4b8 \
     --hash=sha256:81084f3d426f5184f0a0f1dbfe035cc26b66f041d2184559d916a228d856f0d3
     # via -r requirements/common.in
+https://github.com/typeddjango/django-stubs/archive/refs/heads/master.zip \
```

Comment:
We can’t use `master.zip`, since that changes over time. If a Git version is needed, use a specific commit hash.

Reply:
Yes, ideally we use a release if we can. This is currently for development only.

Reply:
What I’m saying is that it’s not working *at all*—not even for development only.

Reply:
I see. Actually, it's not even installing the latest django-stubs-ext properly. Updated the setup commit.

Reply:
Do we want to put `django-stubs-ext` in `common.in`? Without monkey-patching using `django-stubs` specific generics and importing types like `ValuesQuerySet` (exported from `django-stubs-ext`) cause runtime errors. Or, should we limit them all to `TYPE_CHECKING` only and use quoted types?